doc/design/linc_permissions.md

```diff
@@ -1,43 +1,69 @@
 
-https://github.com/orgs/lincbrain/projects/2?pane=issue&itemId=54650571
+### Use case 2
+Set a single Neuroglancer viewer programmatically to visualize multiple assets (e.g. dMRI, HiP-CT) as layers.   See [example notebook](https://github.com/neuroscales/ngtools/blob/main/notebooks/show_tract.ipynb).
 
-https://github.com/neuroscales/ngtools/blob/main/notebooks/show_tract.ipynb
+### Use case 3
+Use case 2 + Load the tractography-generated streamlines (i.e. `trk` file) into memory,
+convert the streamlines into the Neuroglancer skeleton format, and visualize in the Neuroglancer
+viewer as an additional layer.  See [example notebook](https://github.com/neuroscales/ngtools/blob/main/notebooks/show_tract.ipynb).
 
-https://github.com/google/neuroglancer/issues/507
+## Requirements
 
-## Implementation Options
+1. Zarr archives are stored in a private AWS S3 bucket
+2. Provide direct access to the LINC users of the assets on the private S3 bucket using a URI.
 
-#### Issue read-only credentials for S3 bucket. Quick set up.
+## Implemented solution
 
-Does not resolve rendering issue. Would resolve accessing the asset in Jupyter Notebook, but would require flaky Python code.
+- Summary: Integrate read-only credentials for S3 bucket into LINC Archive, LINC CLI.
+ CloudFront distribution with Origin Access Identity control for relevant S3 buckets
 
-#### Integrate read-only credentials for S3 bucket into LINC Archive, LINC CLI
+- S3 Asset Bucket is served via a CloudFront distribution
 
-###### CloudFront distribution with Origin Access Identity control for relevant S3 buckets
+- User navigates to LINC Archive (i.e. `lincbrain.org`) to authenticate. Thereby the user hits a get_presigned_cookie/ API endpoint. In the endpoint response is a valid cookie that allows the user's browser to be able to access the asset via their browser.
 
-- S3 Asset Bucket is served via a CloudFront distribution
-- User hits a get_presigned_cookie/ API endpoint in LINC Archive. In the endpoint response is a valid cookie that allows the user's browser to be able to access the asset via their browser
+Render with hosted Neuroglancer
+- Host Neuroglancer fork on S3 bucket with static site hosting enabled. Create an AWS CloudFront distribution to sit as a verified proxy for permission to many S3 assets at once via presigned cookies.  Provide CNAME for CloudFront Distribution URL to `neuroglancer.lincbrain.org` so that cookies generated by LINC Archive (also living at a `*.lincbrain.org`) can be shared between the API and the Neuroglancer site.
+- Modify the Neuroglancer source code to handle a CloudFront asset path (i.e. `neuroglancer.lincbrain.org` since the domains of the presigned cookie are limited to `*.lincbrain.org` domains due to same-site requirements for cookies) that is similar to any S3 asset path (i.e. `s3://`).
+- Cookies would get passed in request to get asset
```

Comment:
```suggestion
- Cookies would get passed in request to get asset
- Update CloudFront cache policy time to live (TTL) setting to 1 second.  Since LINC assets can be overwritten with the same filename, the CloudFront cache layer would not know to update the cache contents.  This updated policy will allow for the updated asset to be displayed correctly by Neuroglancer.
```